src/test/java/de/retest/web/it/InteractionPseudoElementIT.java

```diff
@@ -0,0 +1,61 @@
+package de.retest.web.it;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import de.retest.recheck.Recheck;
+import de.retest.recheck.RecheckImpl;
+import de.retest.recheck.RecheckOptions;
+import de.retest.recheck.persistence.SeparatePathsProjectLayout;
+import de.retest.web.testutils.PageFactory;
+import de.retest.web.testutils.WebDriverFactory;
+
+class InteractionPseudoElementIT {
+
+	Recheck re;
+	WebDriver driver;
+
+	@BeforeEach
+	void setUp( @TempDir final Path temp ) {
+		re = new RecheckImpl( RecheckOptions.builder() //
+				.disableReportUpload() //
+				.projectLayout( new SeparatePathsProjectLayout( temp.resolve( "state" ), temp.resolve( "reports" ) ) ) //
+				.setIgnore( "interaction-pseudo-elements.filter" ) //
+				.build() );
+
+		driver = WebDriverFactory.driver( WebDriverFactory.Driver.CHROME );
+	}
+
+	@AfterEach
+	void tearDown() {
+		driver.quit();
+		re.cap();
+	}
+
+	@Test
+	void ghost_pseudo_elements_should_not_appear_because_of_css_changes() {
+		re.startTest();
+
+		driver.get( PageFactory.toPageUrlString( "pseudo/index.html" ) );
+
+		final WebElement withoutPseudo = driver.findElement( By.id( "link-without-pseudo" ) );
+		re.check( driver, "click" );
+		withoutPseudo.click();
+		re.check( driver, "click" ); // This should not capture any differences
+
+		assertThatThrownBy( re::capTest ).hasMessageEndingWith(
+				"2 check(s) in 'de.retest.web.it.InteractionPseudoElementIT' found the following difference(s):\n"
+						+ "Test 'ghost_pseudo_elements_should_not_appear_because_of_css_changes' has 1 difference(s) in 2 state(s):\n"
+						+ "click resulted in:\n"
+						+ "\tNo Golden Master found. First time test was run? Created new Golden Master, so don't forget to commit..." );
```

Comment:
I'm a bit torn here, because I would want the test to have no dependencies (i.e. prior test runs to create the Golden Master) and have an easy check (i.e. simply have the test run without assertions).

Since I have created the test to run from scratch, I must create a first Golden Master (first "click"), which always creates a "No Golden Master found" (we currently have no way of avoiding that). The actual test is the second "click" which will use the first Golden Master to compare against.

The assertion checks for:

1. Check by `hasMessageEndingWith`, thus no differences can follow.
1. Exactly 2 checks were performed.
2. By the difference message: "has 1 difference(s)", which will ever only be "No Golden Master found".

I hoped that this is clear, and maybe with an explanation it will be. However, I must admit, that for this case, it might be beneficial to commit the Golden Master (which we know will cause problems later on, as seen by the other failing tests) so that we can simply check: `assertThatCode( re::capTest ).doesNotThrowAnyException()` or simply `re.capTest()`.

Reply:
I guess in that case I would prefer committing the Golden Master. "which we know will cause problems later on": What problems are you referring to specifically? We probably should try to address these more directly instead...

Reply:
> What problems are you referring to specifically?

Those described by my [comment](https://github.com/retest/recheck-web/pull/612#issuecomment-658734547). I will need to create a issue for that, explaining the problems in more depth, so that we can discuss that there.

Reply:
"as the some tests expect certain differences to be present in a Golden Master (and thus should not be accepted)" That isn't the case here, so I don't see the problem in this case—but of cause I am open :-)

Reply:
Let's discuss the different types of tests separately, once I have created the issue. For now, I will change the test to commit the Golden Master, as we both agree that this would be more readable.

Reply:
Great, thanks a lot!